openedx/license-manager
# feat: Add renewed_to field to the License model

* updates the renewal processing ADR to reflect how we'll use this field.
* deletes the `tranferred-renewal` license status choice.

These fields allow us to know the pre-renewal status of a license after the renewal is processed (e.g. was it "assigned" or "activated" before the renewal happened?).  It will also let us directly reference licenses that are related to each other through the renewal of related subscription plans.

Testing instructions
* Inide an `app-shell` run migrations with: `./manage.py migrate`
* Login into local license-manager as an admin and visit http://localhost:18170/admin/subscriptions/license/
* Click on a license.
* Notice the new fields `renewed_to` and `renewed_from` (which is the reverse of the `renewed_to` FK).
